pkg/cpi/mockingUtils.go

```diff
@@ -33,6 +29,25 @@ func GetCPIFunctionMockResponse(functionName, testType string) (*http.Response,
 		}
 		return &res, errors.New("Internal Server Error")
 
+	case "IntegrationArtifactUpdateConfiguration":
```

Comment:
This case has nothing to do with this PR and should not be part of it.

Reply:
removed